### ISSUE TITLE
Duration on each page

### ISSUE THREAD

**Issue by emschwartz** [electerious/Ackee#140]
Would it be possible to show the average duration on each page? Each of the pages on the site I'm tracking has different expected user behavior so I would be curious to see how long people spend on the front page versus the real content of the site.

Also, thanks for creating this project!

**Comment by electerious** [comment]
It's not planned, but it sounds like a good idea. Maybe something for a future version, but it has a low priority for now.

Please upvote if anyone else is interested in this a feature 👍

**Comment by emschwartz** [comment]
Fair enough. In the meantime, I created separate "domains" for each page so that Ackee would track them independently
